modules/microshift-manifests-upgrade-scenario-ostree.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-upgrade-scenario-ostree_{context}"]
+== Manifest upgrade scenario for OSTree systems
+Use the following procedure in the upgrade scenario for OSTree systems where some resources need to be removed while retaining others to preserve data.
+
+[IMPORTANT]
+====
+For OSTree systems, the `delete` subdirectories are read only.
+====
+
+.Procedure
+
+. Identify the manifest that needs updating.
+. Create new manifest for resource deletion that needs to be placed in the `delete` subdirectories.
+. Create new manifest that needs to be applied in the manifest path. link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{ocp-version}/html/running_applications/applications-with-microshift#microshift-applying-manifests-example_applications-microshift[Using manifests example] to create new manifests using the example.
```

Comment:
```suggestion
. Create a new manifest to apply in the manifest path. link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{ocp-version}/html/running_applications/applications-with-microshift#microshift-applying-manifests-example_applications-microshift[Using manifests example] to create new manifests using the example.
```
See previous comment about using `access.redhat.com` links. Also, this sentence containing the cross reference seems to be missing a word at the beginning.

Reply:
done as per comment https://github.com/openshift/openshift-docs/pull/83246#discussion_r1817012102.